modules/gapi/test/serialization/gsrlzbackend.hpp

```diff
@@ -0,0 +1,77 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_GSRLZBACKEND_HPP
+#define OPENCV_GAPI_GSRLZBACKEND_HPP
+
+#include <map>                // map
+#include <unordered_map>      // unordered_map
+#include <tuple>              // tuple
+#include <ade/util/algorithm.hpp> // type_list_index
+
+#include <opencv2/gapi/garg.hpp>
+#include <opencv2/gapi/gproto.hpp>
+#include "gsrlzkernel.hpp"
+#include "core.hpp"
+#include "imgproc.hpp"
+
+#include "api/gorigin.hpp"
+#include "backends/common/gbackend.hpp"
+#include "compiler/gislandmodel.hpp"
+#include "compiler/passes/passes.hpp"
+
+namespace cv { namespace gimpl {
```

Comment:
please use another namespace for this

Reply:
maybe `opencv_test { serialization { impl {`

Reply:
Done